automation/run-tests.sh

```diff
@@ -258,8 +258,11 @@ then
 fi
 
 docker_exec 'while ! systemctl is-active dbus; do sleep 1; done'
+docker_exec 'systemctl start systemd-udevd
```

Comment:
Is this really needed?

Reply:
Also, if it is needed, the CentOS images should be fixed to require systemd-udevd as well.

Reply:
On CentoOS it seems to be part of the systemd package and it comes enabled.
On Fedora, it was a requirement of `dracut`, but now they seems to have dropped the requirement.

So yes, it is needed otherwise it does not work.

Reply:
Correction: `dracut` itself got dropped from the base image, that is why `systemd-udev` is now missing, not because of requirement changes in `dracut`.